challenges/objects-arrays.js

```diff
@@ -85,31 +121,38 @@ The zoo wants to display both the scientific name and the animal name in front o
 
 */
 const animalNames = [];
-console.log(animalNames);
 
+zooAnimals.forEach(function(zooAnimal){console.log(`Name: ${zooAnimal.animal_name}, Scientific: ${zooAnimal.scientific_name}.`)})
+console.log(animalNames);
 /* Request 2: .map()    
 
 The zoos need a list of all their animal's names (names only, not scientific) converted to lower case.  Create a new array named lowerCase and map over each name to convert them all to lower case.  Log the resut.
 
 */
 
 const lowerCase = [];
+lowerCase.push(zooAnimals.map(function(zooAnimal){return zooAnimal.animal_name.toLowerCase()})); 
+
 console.log(lowerCase); 
 
 /* Request 3: .filter() 
 
 The zoos are concenred about animals with a lower population count. Find out which animals have a population less than 5.
 
 */
-const largerPopulation = [];
+const largerPopulation = []; //is is a trick to call the variable largePopulation and ask about small populations?
+
+largerPopulation.push(zooAnimals.filter((zooAnimal)=>zooAnimal.population < 5));
```

Comment:
Similar to the above lowerCase issue, here you are creating a new array (with the `map` array method) and then pushing that newly created array into the largerPopulation array. So you're making an array of objects nested inside of another array, like this: `[ [{'property': 'value'}] ]` when it should just be an array of objects, like this: `[{'property': 'value'}]`.

To resolve this, just assign the newly created array to the largerPopulation variable, instead of pushing the newly created array into the array assigned to the largerPopulation variable. Here is an example (I'm also using arrow notation), please let me know if you have any questions:
```
const largerPopulation = zooAnimals.filter(zooAnimal => (zooAnimal)=>zooAnimal.population < 5)); 
```